src/utils/select-rows.js

```diff
@@ -14,17 +14,20 @@ export function selectRows(rowInfo, options) {
         if(options.rowFilter && options.rowFilter.length > 0) {
             const filterInfos = options.rowFilter;
             filterInfos.forEach(info => {
-                const { field, type, contains } = info;
+                const { field, type, contains, range } = info;
                 const isMultipleFields = Array.isArray(field);
                 if(type === "nominal") {
                     filteredRowInfo = filteredRowInfo.filter(d => d[1][field].toString().toUpperCase().includes(contains.toUpperCase()));
                 } else if(type === "quantitative") {
-                    // TODO: Better deal with quantitative data. Need to update Wrapper options for this.
-                    // refer vega filter, such as lt: https://vega.github.io/vega-lite/docs/filter.html
+                    const [minCutoff, maxCutoff] = range;
                     if(isMultipleFields) {
-                        //...
+                        filteredRowInfo = filteredRowInfo.filter(d => {
+                            let sum = 0;
+                            field.forEach(f => sum += d[1][f]);
+                            return sum > minCutoff && sum < maxCutoff;
+                        });
                     } else {
-                        filteredRowInfo = filteredRowInfo.filter(d => d[1][field].toString().includes(contains));
+                        filteredRowInfo = filteredRowInfo.filter(d => d[1][field] > minCutoff && d[1][field] < maxCutoff);
                     }
                 } else if(type === "tree") {
                     filteredRowInfo = filteredRowInfo.filter(d => d[1][field].reduce((a, h, i) => a && (i >= contains.length || h === contains[i]), true));
```

Comment:
It looks like something in this pull request breaks the dendrogram subtree filtering, would you be able to investigate this?

<img width="1353" alt="Screen Shot 2020-03-24 at 12 17 17 PM" src="https://user-images.githubusercontent.com/7525285/77450324-78289100-6dc9-11ea-9ab9-1cbc7daa2eec.png">

<img width="1353" alt="Screen Shot 2020-03-24 at 12 17 24 PM" src="https://user-images.githubusercontent.com/7525285/77450336-7bbc1800-6dc9-11ea-9e8d-594b0ef4f544.png">

Reply:
The bug was based on not considering `"tree"` type field when picking proper filtering `condition` (`contains` vs. `range`). It should work now.

Regarding this, I think we may need to add an additional `condition` for subtree filtering, since `contains` can be misleading.